client/src/components/login/login.component.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './login.styles.scss';
+import loginImage from '../../assets/login.svg';
+
+const LoginComponent = () => {
+    return (
+        <div className="login-container">
+            <div className="base-container">
+                <div className="header">
+                    Login
+                </div>
+                <div className="content">
+                    <div className="image">
+                        <img src={loginImage} />
+                    </div>
+                    
+                    <div className="form">    
+                        <div className="form-group">
+                            <label htmlFor="email">Email</label>
```

Comment:
`htmlFor` uses the `id` of the `input`, not the `name`. Also, you can use the `for` attribute instead.